test/y2storage/boot_requirements_checker_ppc_test.rb

```diff
@@ -83,6 +83,44 @@
       end
     end
 
+    RSpec.shared_examples "PReP + /boot partition" do
```

Comment:
Why is this extracted to a `shared_examples` block if it's actually not shared?

Reply:
See my 'Note' in the pr description. I wasn't sure I'm going to need it.

Reply:
Anyway, does it hurt? The test looks more readable this way, imho.

Reply:
Fine for me. I was just curious